canvas_account_admin_tools/settings/local.py

```diff
@@ -43,22 +43,18 @@
             'course_short_title': 'Selenium Auto Test 101',
             'course_title': 'Selenium Automated Test Course 101',
             'template': 'None',  # No Template
-            'term': '1',
+            'term': 'string:5209',
```

Comment:
Talked offline about the change in term: That's because term=1 doesn't exist anymore